### ISSUE TITLE
Support "Node?"

### ISSUE THREAD

**Issue by domenic** [jsdom/webidl2js#16]
isEqualNode's `(Node? node)` parameter list should convert `undefined`s to `null`s so that impl code doesn't have to check for undefined.

**Comment by Sebmaster** [comment]
https://github.com/jsdom/webidl2js/blob/master/lib/parameters.js#L30 should already test that. Not working?

**Comment by domenic** [comment]
In the generated output I only saw this:

```js
  if (!this || !module.exports.is(this)) {
    throw new TypeError("Illegal invocation");
  }
  if (arguments.length < 1) {
    throw new TypeError("Failed to execute 'isEqualNode' on 'Node': 1 argument required, but only " + arguments.length + " present.");
  }
  const args = [];
  for (let i = 0; i < arguments.length && i < 1; ++i) {
    args[i] = utils.tryImplForWrapper(arguments[i]);
  }
  return utils.tryWrapperForImpl(this[impl].isEqualNode.apply(this[impl], args));
```

**Comment by Sebmaster** [comment]
Fixed by e10432f.
